RNReanimated.podspec

```
@@ -2,22 +2,75 @@ require "json"
+$new_arch_enabled = ENV['RCT_NEW_ARCH_ENABLED'] == '1'
 is_release = ENV['PRODUCTION'] == '1'
-using_hermes = ENV['USE_HERMES'] == nil || ENV['USE_HERMES'] == '1'
 
-folly_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32 -DREACT_NATIVE_MINOR_VERSION=' + config[:react_native_minor_version].to_s
+folly_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32 -DREACT_NATIVE_MINOR_VERSION=' + $config[:react_native_minor_version].to_s
 folly_compiler_flags = folly_flags + ' ' + '-Wno-comma -Wno-shorten-64-to-32'
 boost_compiler_flags = '-Wno-documentation'
-fabric_flags = fabric_enabled ? '-DRCT_NEW_ARCH_ENABLED' : ''
-example_flag = config[:is_reanimated_example_app] ? '-DIS_REANIMATED_EXAMPLE_APP' : ''
+fabric_flags = $new_arch_enabled ? '-DRCT_NEW_ARCH_ENABLED' : ''
+example_flag = $config[:is_reanimated_example_app] ? '-DIS_REANIMATED_EXAMPLE_APP' : ''
 version_flag = '-DREANIMATED_VERSION=' + reanimated_package_json["version"]
 debug_flag = is_release ? '-DNDEBUG' : ''
-ios_min_version = config[:react_native_minor_version] >= 73 ? '10.0' : '9.0'
+
+module LegacyModule
+  def self.install_modules_dependencies(s)
+    using_hermes = ENV['USE_HERMES'] == nil || ENV['USE_HERMES'] == '1'
+    s.dependency "React-Core"
+    if $new_arch_enabled
+      s.dependency "React-RCTFabric"
+      s.dependency "React-Codegen"
+    end
+    s.dependency "RCT-Folly"
+    s.dependency "RCTRequired"
+    s.dependency "RCTTypeSafety"
+    s.dependency 'FBLazyVector'
+    if $config[:react_native_minor_version] <= 71
+      s.dependency 'FBReactNativeSpec'
+    end
+    s.dependency 'React-Core'
+    s.dependency 'React-CoreModules'
+    s.dependency 'React-Core/DevSupport'
+    if !$config[:is_tvos_target]
+      s.dependency 'React-RCTActionSheet'
+    end
+    s.dependency 'React-RCTNetwork'
+    s.dependency 'React-RCTAnimation'
+    s.dependency 'React-RCTLinking'
+    s.dependency 'React-RCTBlob'
+    s.dependency 'React-RCTSettings'
+    s.dependency 'React-RCTText'
+    s.dependency 'React-RCTImage'
+    s.dependency 'React-Core/RCTWebSocket'
+    s.dependency 'React-cxxreact'
+    s.dependency 'React-jsi'
+    s.dependency 'React-jsiexecutor'
+    s.dependency 'React-jsinspector'
+    s.dependency 'Yoga'
+    s.dependency 'DoubleConversion'
+    s.dependency 'glog'
+    if using_hermes && !$config[:is_tvos_target] && $config[:react_native_minor_version] >= 70
+      s.dependency 'React-hermes'
+      s.dependency 'hermes-engine'
+    end
+    s.dependency 'React-callinvoker'
+    if $config[:react_native_minor_version] >= 72 && !$new_arch_enabled
+      s.dependency 'React-RCTAppDelegate'
+    end
+  end
+
+  def self.get_ios_min_version()
+    if $config[:react_native_minor_version] <= 72
+      return '9.0'
+    end
+    # 0.73+ supports iOS 13.4+, https://github.com/facebook/react-native/pull/39478
+    return '13.4'
```

Let's always use `>=` when comparing RN versions starting from the newest to oldest
```suggestion
    if $config[:react_native_minor_version] >= 73
      return '13.4'
    else
      return '9.0'
    end
```